sparkplug/config/timer.py

```diff
@@ -0,0 +1,49 @@
+
+import datetime, traceback
+from sparkplug.logutils import LazyLogger
+import sparkplug.config.timereporter
+
+_log = LazyLogger(__name__)
+
+##################################################
+
+class Timer( object ):
+    def __init__( self, callback, timers=[] ):
+        self._timers = tuple( sparkplug.config.timereporter.registry[t] for t in timers if t in sparkplug.config.timereporter.registry )
```

Comment:
```
from sparkplug.config import timereporter
self._timers = tuple(timereporter.registry.values())
```

Reply:
Hi Dean, it's not exactly the same semantics.  It's essentially a set intersection operation, but using two lists instead of sets.